modules/standard/OS.chpl

```diff
@@ -956,6 +956,25 @@ module OS {
 
   } // end POSIX
 
+/* A type storing an error code or an error message.
+   An :type:`errorCode` can be compared using == or != to a
+   :type:`CTypes.c_int` or to another :type:`errorCode`. An :type:`errorCode`
+   can be cast to or from a :type:`CTypes.c_int`. It can be assigned the
+   value of a :type:`CTypes.c_int` or another :type:`errorCode`. In addition,
+   :type:`errorCode` can be checked directly in an if statement like so:
+
+   .. code-block:: chapel
+
+     var err: errorCode;
+     if err then do writeln("err contains an error, ie err != ENOERR");
+     if !err then do writeln("err does not contain an error; err == ENOERR");
+
+   When an :type:`errorCode` formal has default intent, the actual is copied to
```

Comment:
Consider: "The default intent for a formal of type `errorCode` is `const in`? I think that communicates the same thing.